### ISSUE TITLE
How to change my owner private key on bloks.io if connected with ledger nano.

### ISSUE THREAD

**Issue by Rayezor** [tarassh/fairy-wallet#162]
Hello.

My owner key was compromised so I am trying to find a way to change my key or to make a new account and connect it to my ledger. 

I have been unsuccessful in changing my key using different methods as my ledger nano does not let me change the key. I have made a new account on bloks but unable to find out how to connect it to my ledger instead of my old account. Any advice?

Ray.

**Comment by lshaf** [comment]
any luck bro? I'm looking for this also.

**Comment by Rayezor** [comment]
Hey man,

Yeah this happened a while ago but I never got a solution. I was afraid
that I potentially compromised the key from a scam site advertising an
airdrop.

I never lost anything but I decided just to move my Eos out of the wallet
just in case and stopped using the wallet.

Best of luck.

**Comment by lshaf** [comment]
Well, I just found out it's bug for EOS 1.4.2 I suppose.
I just tested it using ledgerHQ repository here https://github.com/LedgerHQ/app-eos .
Have no idea what's wrong with that. I already contacted both ledger support and put issue on original repository.

Hope they will reply soon and get it fixed.

**Comment by Rayezor** [comment]
If you find a way to do. I'd appreciate it if you could let me know.

Good luck getting to the moon.

**Comment by lshaf** [comment]
I'm pretty sure they will update EOS app for it in order to fix it.
So just wait till it happen or maybe I will inform you when it's available in here.

**Comment by lshaf** [comment]
@Rayezor good news bro, they update it week ago and yet no public release until I bough new nano s and build it on my own then load it to nano s and then it works.

I've asked them to patch live version with the new one and they said ASAP.
I don't know about ETA but if you want it right away, you can do the same as me.

**Comment by Rayezor** [comment]
Awesome, thanks for the update! So when the update comes in will I be able
to change my private key address?

Ray.

**Comment by lshaf** [comment]
@Rayezor  yes of course, I've tried and successfully change both my owner and active key.
